src/components/Footer/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import 'styled-components/macro';
+import { media } from 'components/breakpoints';
+import { colors, GU } from 'components/theme';
+
+import { ReactComponent as SelectorSvg } from 'assets/icons/selector.svg';
+import { ReactComponent as DiscrodSvg } from 'assets/icons/discord.svg';
```

Comment:
Hmm there seems to be a typo on `DiscrodSvg` 😛

Reply:
Fixed in 26fc03638439a3ab83c22444f4b60af2254954e8